pac.go

```diff
@@ -27,7 +30,18 @@ var (
 func EnsureHelperToolPresent(path string, prompt string, iconFullPath string) (err error) {
 	mu.Lock()
 	defer mu.Unlock()
-	pacBytes, err := Asset("pac")
+	assertName := "pac"
+	// Load different binaries for 32bit and 64bit Windows respectively.
+	if runtime.GOOS == "windows" {
+		suffix := "_386.exe"
+		// https://blogs.msdn.microsoft.com/david.wang/2006/03/27/howto-detect-process-bitness/
+		if strings.EqualFold(os.Getenv("PROCESSOR_ARCHITECTURE"), "amd64") ||
+			strings.EqualFold(os.Getenv("PROCESSOR_ARCHITEW6432"), "amd64") {
+			suffix = "_amd64.exe"
```

Comment:
Any reason not to follow the convention established for linux where each binary is in its own folder?